calisphere/es_cache_retry.py

```diff
@@ -214,7 +214,13 @@ def query_encode(query_string: str = None,
     if query_string:
         es_query = [{
             "query_string": {
-                "query": query_string
+                "query": query_string,
+                "fields": [
+                    "title^2",
+                    "*"
```

Comment:
I wonder if there is any value in being explicit about which fields we want to search. 

There are definitely fields we don't need to search, like `mapper_type`, `fetcher_type`, `repository_id`, all the urls, etc. I'd also wonder on things like `rights`, `rights_note` and even `repository_name`. 
Search less fields on every search is definitely a performance improvement.

Alternatively, if we dumb all searchable fields into a 'text' bucket field, then we only search the bucket plus the other fields we want to boost.

Reply:
I believe that it's currently only searching across the `text` type fields, since this is a full-text search, and `keyword` fields can't be full-text searched by definition. So we're not searching mapper_type, fetcher_type, etc: 

https://github.com/ucldc/rikolti/blob/main/record_indexer/index_templates/record_index_config.py

We might want to exclude some of the metadata fields you mention above though. (I was trying to keep parity with what we currently have on production).